From 2770dffa54f73cab801c96ff85f34dbe8abc5e47 Mon Sep 17 00:00:00 2001
From: Matt Jolly <kangie@gentoo.org>
Date: Mon, 14 Oct 2024 15:58:37 +1000
Subject: [PATCH] games-server/mednafen-server: Explicitly include
 `ax_cflags_gcc_option.m4`

We need to call `eautoreconf` in `src_prepare` to
fix some C99/C23 issues in the shipped configure file.

In order to do this, we need to explicitly include
`ax_cflags_gcc_option.m4` in `configure.ac`. I'm
not sure how this worked previously but it doesn't
anymore.

---
 configure.ac | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configure.ac b/configure.ac
index 0da3ab4..483f837 100644
--- a/configure.ac
+++ b/configure.ac
@@ -8,6 +8,7 @@ AC_CANONICAL_HOST
 AC_CANONICAL_TARGET
 AM_INIT_AUTOMAKE([mednafen_server], $MEDNAFEN_SERVER_VERSION)
 m4_ifdef([AM_SILENT_RULES], [AM_SILENT_RULES])
+m4_include([m4/ax_cflags_gcc_option.m4])
 AM_CONFIG_HEADER([include/config.h:include/config.h.in])
 
 AC_PROG_CC
-- 
2.46.2
